rubocop/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ruby:alpine
```

Comment:
Consider using the same version in both docker files. Saves someone having to potentially download two different images.